.github/workflows/close-inactive-issues.yml

```diff
@@ -0,0 +1,60 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *" # once a day (1:30 UTC)
+
+jobs:
+  close-issues-without-assignee:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 7
+          days-before-issue-close: 3
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 7 days with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for 3 days since being marked as stale."
+          remove-issue-stale-when-updated: true
+          exempt-all-issue-milestones: true # issues with assigned milestones will be ignored
+          exempt-all-issue-assignees: true # issues with assignees will be ignored
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          debug-only: true
+
+  close-issues-with-assignee:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 28
+          days-before-issue-close: 7
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 28 days with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for 7 days since being marked as stale."
+          remove-issue-stale-when-updated: true
+          exempt-all-issue-milestones: true # issues with assigned milestones will be ignored
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          debug-only: true
+
+  close-inactive-pull-requests:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-pr-label: "stale"
+          stale-pr-message: "This pull request is stale because it has been open for 28 days with no activity."
+          close-pr-message: "This pull request was closed because it has been inactive for 7 days since being marked as stale."
+          days-before-pr-stale: 28
```

Comment:
considering we're still in a phase where the code base evolves pretty fast, I'd say 7+7 for stale and close would be acceptable. A PR that has been neglected for a month is likely abandoned anyways.